smashtheque/smashtheque.py

```diff
@@ -265,6 +265,14 @@ async def find_tournament_by_id(self, tournament_id):
             tournament = await response.json()
             return tournament
 
+    async def find_tournament_by_discord_id(self, discord_id):
+        request_url = f"{self.api_url('recurring_tournaments')}.json?by_discord_guild_discord_id={discord_id}"
+        async with self._session.get(request_url) as response:
+            if response.status == 404:
+                return None
+            tournament = await response.json()
+            return tournament[0]

```

Comment:
Il faudrait gérer le cas où le tableau est vide, ici, et retourner None, non ? Sinon ça fera un `IndexError: list index out of range`

Reply:
Ok, got it !